contracts/settling_game/utils/constants.cairo

```diff
@@ -63,30 +63,52 @@ namespace SHIFT_41:
     const _5 = 2 ** 164
 end
 
-const TRUE = 1
-const FALSE = 0
+# -----------------------------------
+# Resources + vault
+# -----------------------------------
+const MAX_DAYS_ACCURED = 2  # max amount of days that can accure without claiming
+const BASE_RESOURCES_PER_DAY = 250  # base resource production per day cycle
+const WONDER_RATE = BASE_RESOURCES_PER_DAY / 10  # resources generated by wonders set at 10% of regular realm
 
-# SETTLING
-const VAULT_LENGTH = 7  # days
-const DAY = 1800  # day cycle length
+const VAULT_LENGTH = 7  # length of day cycles
+const DAY = 1800  # day cycle length in secondary
 const VAULT_LENGTH_SECONDS = VAULT_LENGTH * DAY  # vault is always 7 * day cycle
 
-# CRYPTS
-const RESOURCES_PER_CRYPT = 1  # We only generate one resource per crypt (vs up to 7 per realm)
-const LEGENDARY_MULTIPLIER = 10  # Legendary maps generate 10x resources as non-egendat
+const BASE_LORDS_PER_DAY = 25  # base lords generated by goblins
 
-# PRODUCTION
-const BASE_RESOURCES_PER_DAY = 100
-const BASE_LORDS_PER_DAY = 25
+const WORK_HUT_COST_IN_BP = 5  # 1/5 of resource output
+const WORK_HUT_COST = BASE_RESOURCES_PER_DAY / WORK_HUT_COST_IN_BP  # 1/5 of resource output
+const WORK_HUT_OUTPUT = BASE_RESOURCES_PER_DAY / 10
```

Comment:
Note - we need to be careful here with adjusting the values, as not to have the `/` operation produce an unwanted result.

Reply:
yes, I had that thought - overflow potential easy, but is is very handy having them all here. Perhaps we just make a new lib with SafeMath getters, which just pulls these